CONTRIBUTING.md

```diff
@@ -72,6 +72,60 @@ pub struct Responder {
 ## Test Coverage
 Tests should be provided to cover both positive and negative conditions. Tests should cover both the proper execution as well as all the covered error paths. PR with no proper test coverage will not be merged.
 
-## Signing Commits
+## Git conventions 
+
+### Commits, descriptions, and long descriptions
+
+- Changes must be split logically in commits, such that a commit is self-contained. 
+- In general terms, all commits need to pass the test suite. There may be some exceptions to this rule if the change you are working on touches several components of the codebase and it makes more sense to split the change by component (or group of components).
+- Commit descriptions need to be short and explanatory. If we are, for instance, adding an RPC command to the backend, "Adds command X to the backend" will be a good short description, "Add command" or "Fix #123" where #123 is an issue referencing this feature **IS NOT**.
```

Comment:
I think what's referred to as `Commit descriptions` here are actually called `Commit messages`.

Reply:
I was checking some external sources and felt like there was no consensus on this, given both the short and long messages are called messages 😅 

I think we could settle it, conceptually, as Commit summaries (or titles) and commit descriptions: https://www.gitkraken.com/learn/git/best-practices/git-commit-message#git-commit-message-structure